sql/2022/interop/color_spaces.sql

```diff
@@ -0,0 +1,200 @@
+#standardSQL
+CREATE TEMPORARY FUNCTION getP3Usage(css STRING)
```

Comment:
I understand there are other color spaces, such as HSL, HSLA, LAB and LCH. https://wpt.fyi/results/css/css-color?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-color 

Are we looking at all these or only p3? (I'm not very knowledgeable on this topic TBF)

Reply:
I'm not sure either, I left a comment for @bkardell about this in the Slack channel.

@bkardell what do you envision this query looking like?